rocky/rocky/urls.py

```diff
@@ -72,6 +78,9 @@
         PrivacyStatementView.as_view(),
         name="privacy_statement",
     ),
+    path("tasks/", AllBoefjesTaskListView.as_view(), name="all_task_list"),
```

Comment:
Drop the trailing / for consistency with other routes

Reply:
Consistency with which routes? There does not seem to be consistency.

Reply:
I mistakenly assumed we did not have routes with trailing /'es.